restore the possibility to do "make package/<name>/install" for updating the nfs root

diff --git a/include/package-ipkg.mk b/include/package-ipkg.mk
--- a/include/package-ipkg.mk
+++ b/include/package-ipkg.mk
@@ -6,6 +6,21 @@
 #
 
 include $(INCLUDE_DIR)/feeds.mk
+OPKG:= \
+  IPKG_TMP=$(TMP_DIR)/ipkg \
+  IPKG_INSTROOT=$(TARGET_DIR) \
+  IPKG_CONF_DIR=$(STAGING_DIR)/etc \
+  IPKG_OFFLINE_ROOT=$(TARGET_DIR) \
+  $(STAGING_DIR_HOST)/bin/opkg \
+	--offline-root $(TARGET_DIR) \
+	--force-depends \
+	--force-overwrite \
+	--force-postinstall \
+	--force-maintainer \
+	--force-downgrade \
+	--add-dest root:/ \
+	--add-arch all:100 \
+	--add-arch $(if $(ARCH_PACKAGES),$(ARCH_PACKAGES),$(BOARD)):200
 
 # invoke ipkg-build with some default options
 IPKG_BUILD:= \
@@ -79,6 +94,7 @@ ifeq ($(DUMP),)
     PDIR_$(1):=$(call FeedPackageDir,$(1))
     IPKG_$(1):=$$(PDIR_$(1))/$(1)_$(VERSION)_$(PKGARCH).ipk
     IDIR_$(1):=$(PKG_BUILD_DIR)/ipkg-$(PKGARCH)/$(1)
+    INFO_$(1):=$(IPKG_STATE_DIR)/info/$(1).list
     KEEP_$(1):=$(strip $(call Package/$(1)/conffiles))
 
     ifeq ($(BUILD_VARIANT),$$(if $$(VARIANT),$$(VARIANT),$(BUILD_VARIANT)))
@@ -93,6 +109,7 @@ ifeq ($(DUMP),)
         ifeq ($(CONFIG_PACKAGE_$(1)),y)
           .PHONY: $(PKG_INSTALL_STAMP).$(1)
           compile: $(PKG_INSTALL_STAMP).$(1)
+          install: $$(INFO_$(1))
           $(PKG_INSTALL_STAMP).$(1):
 			if [ -f $(PKG_INSTALL_STAMP).clean ]; then \
 				rm -f \
@@ -194,6 +211,10 @@ ifeq ($(DUMP),)
 	$(IPKG_BUILD) $$(IDIR_$(1)) $$(PDIR_$(1))
 	@[ -f $$(IPKG_$(1)) ]
 
+    $$(INFO_$(1)): $$(IPKG_$(1))
+	@[ -d $(TARGET_DIR)/tmp ] || mkdir -p $(TARGET_DIR)/tmp
+	$(OPKG) install $$(IPKG_$(1))
+	$(if $(filter-out essential,$(PKG_FLAGS)),for flag in $(filter-out essential,$(PKG_FLAGS)); do $(OPKG) flag $$$$flag $(1); done,$(OPKG) flag ok $(1))
     $(1)-clean:
 	rm -f $$(PDIR_$(1))/$(1)_*
 
